docs/drivers/vsphere.md

```diff
@@ -25,7 +25,7 @@ Options:
 -   `--vmwarevsphere-vcenter-port`: vSphere Port for vCenter.
 -   `--vmwarevsphere-network`: Network where the Docker VM will be attached.
 -   `--vmwarevsphere-datastore`: Datastore for Docker VM.
--   `--vmwarevsphere-datacenter`: Datacenter for Docker VM.
+-   `--vmwarevsphere-datacenter`: Datacenter for Docker VM (must be set to ha-datacenter when connecting to a single host).
```

Comment:
Can you please demarcate the "ha-datacenter" strings with a backtick (`) or quotes please?  e.g.:

> (must be set to `ha-datacenter` when connecting to a single host)